frontend/src/context/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import reducer, { ReducerState, ReducerAction } from "./reducer";
+import initialState from "./initialState";
+import { Status } from "./reducer";
+
```

Comment:
Ok this requires explaining.

In react context is a global state. This file is meant to initialize and scaffold a shared global state which I didn't quite get to using.

AppContext is an object containing a state object and dispatch array. State is state, dispatch is an array of dispatch/actionCreators methods.

The Provider component would be wrapped at root and all children components would then `useContext(appContext)` and have acess to what ever state is stored in context.